### PR TITLE
Adds some extra text to the manuals when they're loading

### PR DESCRIPTION
**Pull request by Jujumatic**
Simple enough, just added an extra "please wait" text to manuals while they're loading because I found out they actually work even on hippiestation right now, it's just that it takes a few seconds to load the pages. so i guess i always thought they were just broken.

also i accidentally dirtied this branch with the megaseed stuff but that was probably going through anyway iirc

### CODE REVIEW

File: code/modules/hydroponics/biogenerator.dm [hunk 1]
```diff
@@ -16,8 +16,8 @@
 	var/productivity = 0
 	var/max_items = 40
 	var/datum/techweb/stored_research
-	var/list/show_categories = list("Food", "Botany Chemicals", "Leather and Cloth")
```
Comment (Pyko1) [inline code/modules/hydroponics/biogenerator.dm]: should not be part of this PR

File: hippiestation/code/modules/research/designs/biogenerator_designs.dm [hunk 2]
```diff
@@ -216,4 +216,15 @@
 	build_type = BIOGENERATOR
 	materials = list(MAT_BIOMASS = 100)
 	build_path = /obj/item/melee/baseball_bat
-	category = list("initial","Leather and Cloth")
```
Comment (Pyko1) [inline hippiestation/code/modules/research/designs/biogenerator_designs.dm]: should not be part of this PR

**Comment by Jujumatic** [comment]
fixed and undirtied

**Comment by Pyko1** [comment]
sure +1

**Review by JohnGinnane — APPROVED** [review]
